examples/gallery/images/grdclip.py

```diff
@@ -0,0 +1,35 @@
+"""
+Clipping grid values
+--------------------
+The :meth:`pygmt.grdclip` method allows to clip defined ranges of grid values.
+In the example shown below we set all elevation values (grid points) smaller
+than 0 m (in general the bathymetric part of the grid) to a common value of
+-2000 m via the ``below`` parameter.
+"""
+
+import pygmt
+
+fig = pygmt.Figure()
+
+# Define region of interest around Iceland
+region = [-28, -10, 62, 68]
+
+# Load sample grid (3 arc minute global relief) in target area
+grid = pygmt.datasets.load_earth_relief(resolution="03m", region=region)
+
+# Plot original grid
+fig.basemap(region=region, projection="M12c", frame=["f", '+t"original grid"'])
+fig.grdimage(grid=grid, cmap="oleron")
+
+# Shift plot origin for second map 12.5 cm in x direction
+fig.shift_origin(xshift="12.5c")
```

Comment:
In case you don't know it, we can use **h** in Y shifts and **w** in X shifts, in which **h** and **w** means *height* and *width* of the previous plot. 
```suggestion
# Shift plot origin of the second map by "width of the first map + 0.5 cm"
fig.shift_origin(xshift="w+0.5c")
```

Reply:
> 
> 
> In case you don't know it, we can use **h** in Y shifts and **w** in X shifts, in which **h** and **w** means _height_ and _width_ of the previous plot. See https://docs.generic-mapping-tools.org/latest/cookbook/options.html#plot-positioning-and-layout-the-x-y-options for detailed explanations. Feel free to make more changes to the comment to better explain the `xshift` argument.

Indeed, I didn't know that so far, thanks @seisman.